### ISSUE TITLE
Broken table when ETF requested

### ISSUE THREAD

**Issue by Tim55667757** [Tim55667757/TKSBrokerAPI#38]
```
$ tksbrokerapi --prices IBM MSFT GOOGL UNKNOWN_TICKER POSI YNDX TGLD --output some-prices.md
TKSBrokerAPI.py     L:977  WARNING [2022-07-27 00:21:57,776] Instrument [UNKNOWN_TICKER] not in list of available instruments for current token!
TKSBrokerAPI.py     L:1018 INFO    [2022-07-27 00:21:58,048] Only unique instruments are shown:
# Actual prices at: [2022-07-26 21:21 UTC]

| Ticker       | FIGI         | Type       | Prev. close | Last price  | Chg. %   | Day limits min/max  | Actual sell / buy   | Curr.
|--------------|--------------|------------|-------------|-------------|----------|---------------------|---------------------|------
| IBM          | BBG000BLNNH6 | Shares     |      128.08 |      128.36 |   +0.22% |      126.7 / 130.02 |     127.67 / 128.65 | usd
| MSFT         | BBG000BPH459 | Shares     |      251.90 |      252.14 |   +0.10% |     248.74 / 254.96 |     252.01 / 252.14 | usd
| GOOGL        | BBG009S39JX6 | Shares     |      105.02 |      108.34 |   +3.16% |      97.24 / 118.76 |     108.36 / 108.37 | usd
| POSI         | TCS00A103X66 | Shares     |      910.00 |      910.00 |    0.00% |      533.2 / 1243.6 |           N/A / N/A | rub
| YNDX         | BBG006L8G4H1 | Shares     |     2136.00 |     2136.00 |    0.00% |     1672.6 / 2268.4 |           N/A / N/A | rub
| TGLD         | BBG222222222 | Etfs       |        0.07 |        0.07 |    0.00% | 0.06080000000000001 / 0.0815 |           N/A / N/A | usd

TKSBrokerAPI.py     L:1024 INFO    [2022-07-27 00:21:58,052] Price list for all instruments saved to file: x:\work\projects\PriceGenerator\some-prices.md
```

**Comment by Tim55667757** [comment]
Fixed since v1.3.*

**Comment by Tim55667757** [comment]
Debug build for this issue: tksbrokerapi 1.3.dev67.
PyPI: https://pypi.org/project/tksbrokerapi/1.3.dev67/
